.dockerignore

```diff
@@ -1,4 +1,3 @@
 build/
 *.egg-info/
 venv/
-.git/
```

Comment:
This is ok to remove because the Dockerfile that gets pushed to ghcr.io no longer copies in the current working directory. Rather, it does a pip install.